tests/integration/frameworks/yii/test_basic_cli.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should name Yii2 cli transactions
+*/
+
+/*EXPECT_METRICS_EXIST
+Supportability/framework/Yii2/detected
+OtherTransaction/Action/test-integration-cli-action
+*/
+
+/*EXPECT_ERROR_EVENTS null */
+
+/* WordPress mock app */
```

Comment:
This should've been Yii 🙂 but no bigie, thanks for adding the tests for me!

Reply:
Good catch!

Reply:
Addressed with d059d1ccf7d83098043a3d08b5423c310f4a23be.